src/views/InvolvementProfile/components/Membership/components/AdminCard/index.js

```diff
@@ -52,10 +62,7 @@ const AdminCard = ({ createSnackbar, isSiteAdmin, involvementDescription, onAddM
   };
 
   const handleAddMember = async () => {
-    let formattedUsername = username.toLowerCase();
-    if (!formattedUsername.includes('@gordon.edu')) {
-      formattedUsername = formattedUsername.replace('@gordon.edu', '');
-    }
+    let formattedUsername = stripDomain(username.toLowerCase());
```

Comment:
I don't think the `.toLowerCase()` is necessary here - it was needed to match `'@gordon.edu'`, but since `stripDomain` handles that, I think you can leave the casing as-is without any issues, right?

Reply:
I was just thinking for consistent uploads to the API